client/app/bundles/shared/components/_colors.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * This file contains the color palette for the Kujali interface.
+ */
+
+$theme-color-1: #6d0839;
```

Comment:
Rename as `$theme-color-primary`? Really though it should be colour 🌈

Reply:
We can call it primary, that's fine too. It's possible in the future that a theme has more than 2 colors, so that's why I just used straight up numbers. In the future, you can then assign variables:
```
$text-highlight: $theme-color-secondary;

mark { color: $text-highlight }
```

"Colour"? Haha, I would say CSS use of "color" makes things more consistent, but, if you really want, I can make the change. 😆

Reply:
I think if there's more than two colours in the future, we should just give them straight up names